src/site/content/en/blog/introducing-aurora/index.md

```diff
@@ -0,0 +1,225 @@
+supports open-source tools and libraries with funding. While we hope that there
+is enough overlap in the problems and solution layers to our efforts above to
+translate to other frameworks and tools, we know we can do more. To that end, we
+want to do our part to ensure libraries and frameworks helping developers
+succeed can thrive. That's one reason we will be continuing to invest in the
+Chrome Framework Fund. To date, it has supported work towards Webpack 5, Nuxt.js
+and performance and improvements to ESLint.
+
+## What has our work unlocked so far?
+
+Our work has been focused on **foundational optimizations** for resources like
+images, JS, CSS, fonts. We've shipped a number of optimizations to improve the
+defaults of multiple frameworks, including:
+
+- An Image component in
+  [Next.js](https://nextjs.org/docs/basic-features/image-optimization) that
+  encapsulates best practices for image loading, followed by a collaboration
+  with [Nuxt.js](https://image.nuxtjs.org/) on the same. Use of this component
+  has resulted in significant improvements to paint times and layout shift
+  (example: 57% reduction in Largest Contentful Paint and 100% reduction in
+  Cumulative Layout Shift on
+  [nextjs.org/give](https://mobile.twitter.com/rauchg/status/1321452444656623616/photo/2)).
+- Automated inlining of CSS for Web Font declarations at build time. This
+  feature has landed in
+  [Angular](https://angular.io/guide/workspace-config#fonts-optimization-options)
+  (Google Fonts) and
+  [Next.js](https://nextjs.org/docs/basic-features/font-optimization) (Google
+  Fonts & Adobe Fonts) resulting in notable improvements to Largest Contentful
+  Paint and First Contentful Paint
+  ([example](https://twitter.com/griefcode/status/1387746148883050496)).
+- Inlining critical CSS using
+  [Critters](https://github.com/GoogleChromeLabs/critters) in both
+  [Angular](https://angular.io/guide/workspace-config#styles-optimization-options)
+  and Next.js to reduce paint times. Resulted in a 20-30 point improvement in
+  Lighthouse performance scores in a typical, large-scale Angular app when this
+  was combined with font CSS inlining feature.
+- Out-of-the-box [ESLint support](https://nextjs.org/docs/basic-features/eslint)
+  in Next.js that includes a custom plugin and shareable configuration to make
+  it easier to catch common framework-specific issues at build-time, resulting
+  in more predictable loading performance.
+- An introduction of a built-in performance relayer in [Create React
+  App](https://create-react-app.dev/docs/measuring-performance/) and
+  [Next.js](https://nextjs.org/docs/advanced-features/measuring-performance) to
+  allow easier insight into page performance through web vitals and other custom
+  metrics.
+- [Granular chunking](/granular-chunking-nextjs/) shipped in
+  Next.js and Gatsby, resulting in reduced bundle sizes while improving caching
```

Comment:
resulting in reduction bundle sizes by upto 70%, while improving caching

Reply:
Updated

Reply:
nit: 30% to 70% (to be more representative)